river/command/map.zig

```diff
@@ -176,3 +182,36 @@ fn parseModifiers(allocator: *std.mem.Allocator, modifiers_str: []const u8, out:
     }
     return modifiers;
 }
+
+// TODO remove once https://github.com/ziglang/zig/issues/447 is implemented
```

Comment:
you can already declare an anonymous struct as the return type of the function if you like:
```zig
fn parseOptionalArgs(args: []const []const u8) struct {
    i: usize,
    release: bool,
} {
   ...
}
```
and then use an anon struct literal for the return: 
```zig
return .{ .i = 42, .release = false };
```
We'll likely want to make this more generic in order to avoid code duplication as more options are added to commands in the future though.

Reply:
I'll remove the comment and leave it like it is. When other commands get arguments too we could thing of a more generic solution.